sdk/simulation/tinst/deriv.h

```diff
@@ -47,16 +47,20 @@ typedef uint8_t se_owner_epoch_t[OWNEREPOCH_SIZE];
 
 /* Derive data for seal key */
 typedef struct {
-    uint16_t          key_name;        /* should always be 'SGX_KEYSELECT_SEAL' */
-    sgx_attributes_t  tmp_attr;
-    sgx_attributes_t  attribute_mask;  /* attribute mask from KEYREQUEST */
-    se_owner_epoch_t  csr_owner_epoch;
-    sgx_cpu_svn_t     cpu_svn;         /* CPUSVN from KEYREQUEST */
-    sgx_isv_svn_t     isv_svn;         /* ISVSVN from KEYREQUEST */
-    sgx_prod_id_t     isv_prod_id;     /* ISV PRODID from SECS   */
-    sgx_measurement_t mrenclave;
-    sgx_measurement_t mrsigner;
-    sgx_key_id_t      key_id;          /* KEYID from KEYREQUEST  */
+    uint16_t             key_name;        /* should always be 'SGX_KEYSELECT_SEAL' */
+    sgx_attributes_t     tmp_attr;
+    sgx_attributes_t     attribute_mask;  /* attribute mask from KEYREQUEST */
+    se_owner_epoch_t     csr_owner_epoch;
+    sgx_cpu_svn_t        cpu_svn;         /* CPUSVN from KEYREQUEST */
+    sgx_isv_svn_t        isv_svn;         /* ISVSVN from KEYREQUEST */
+    sgx_prod_id_t        isv_prod_id;     /* ISV PRODID from SECS   */
+    sgx_config_svn_t     config_svn;      /* CONFIGSVN from KEYREQUEST */
+    sgx_config_id_t      config_id;       /* CONFIGID from SECS      */
+    sgx_isvfamily_id_t   isv_family_id;   /* ISV FAMILYID from SECS  */
+    sgx_isvext_prod_id_t isv_ext_prod_id; /* ISV EXTPRODID from SECS */
+    sgx_measurement_t    mrenclave;
```

Comment:
Our previous derived data are based on SGX1.  From latest SDM, the derived data also includes some other fields, such as misc_select/misc_mask, key_policy.  Do you want to update this to more align with the latest SDM?

Reply:
I have updated the derived data based on the latest SDM.